src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -56,6 +56,9 @@
     @Query("select r from Result r left join fetch r.feedbacks where r.id = :resultId")
     Optional<Result> findByIdWithEagerFeedbacks(@Param("resultId") Long id);
 
+    @Query("select r from Result r left join fetch r.submission left join fetch r.feedbacks where r.id = :resultId")
```

Comment:
shouldn't we use text blocks for new queries from now on?

Reply:
I think we decided to use them only for large queries so that they are more readable. Maybe I got this wrong?

Reply:
Okay, I though @MaisiKoleni wanted to exchange them bit by bit. But I am also not sure if we defined a formatting schema